<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormDlgRuleSnippet</class>
 <widget class="QDialog" name="FormDlgRuleSnippet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>155</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Rule Snippet Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtRuleSnippetName">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Snippet</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="edtRuleSnippet"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>&amp;Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnSave</sender>
   <signal>clicked()</signal>
   <receiver>FormDlgRuleSnippet</receiver>
   <slot>slotBtnSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>FormDlgRuleSnippet</receiver>
   <slot>slotBtnCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotBtnSave()</slot>
  <slot>slotBtnCancel()</slot>
 </slots>
</ui>
